cmd/api/src/api/middleware/middleware.go

```diff
@@ -152,16 +148,23 @@ func ContextMiddleware(next http.Handler) http.Handler {
 	})
 }
 
-func parseUserIP(r *http.Request) (string, error) {
+func parseUserIP(r *http.Request) string {
+	res := ""
 	if ipAddress := r.Header.Get("X-Forwarded-For"); ipAddress != "" {
-		return strings.Split(ipAddress, ",")[0], nil
-	} else if parsedUrl, err := url.Parse(r.RemoteAddr); err != nil {
-		return "", fmt.Errorf("error parsing IP address from RemoteAddr: %s", err)
+		res += "X-Forwarded-For: " + ipAddress + "; "
+	} else {
+		log.Errorf("No data found in X-Forwarded-For")
```

Comment:
I think we'll get a lot of these in CE? Since most of the time it won't have the header

Reply:
This has been changed to a warning level with a little more context in another PR. Shouldn't be too noisy, since most will probably just turn warnings off in most cases (or filter)